lib/v2/learnku/topic.js

```diff
@@ -0,0 +1,71 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const community = ctx.params.community;
+    const category = ctx.params.category || '';
+
+    let url = `https://learnku.com/${community}`;
+    if (category !== '') {
+        url = `https://learnku.com/${community}/c/${category}`;
+    }
+
+    const response = await got({
+        method: 'get',
+        url,
+    });
+
+    const data = response.data;
+    const $ = cheerio.load(data);
+    const list = $('.simple-topic').get();
+    const item = await Promise.all(
+        list.map(async (item) => {
+            const $ = cheerio.load(item);
+            const categoryName = $('.category-name span').text().trim();
+            if (['置顶', '广告'].includes(categoryName)) {
+                return '';
+            }
+            $('.topic-title i').remove();
+            const itemLink = $('.topic-title-wrap').attr('href');
+
+            const title = $('.topic-title').text().trim();
+            const content = await ctx.cache.tryGet(itemLink, async () => {
+                const result = await got.get(itemLink);
+
+                return cheerio.load(result.data);
+            });
+            const article = content('.article-content .content-body').html();
+            const comment = content('#all-comments').html();
+
+            return {
+                title,
+                description: art(path.join(__dirname, 'templates/topic.art'), {
+                    article,
+                    comment,
+                }),
+                category: categoryName,
+                link: itemLink,
+                pubDate: new Date($('.timeago').attr('title')),
```

Comment:
Please use `parseDate()`.
Docs: https://docs.rsshub.app/joinus/pub-date.html

Reply:
Thank you for your review, This problem has been fixed.

Reply:
Thank you very much for reviewing for me. I hope you can review it again sometime. @TonyRL